src/app_service/service/chat/service.nim

```diff
@@ -300,6 +300,11 @@ QtObject:
 
       discard status_chat.deactivateChat(chatId)
 
+      var channelGroupId = chat.communityId
+      if (channelGroupId == ""):
+        channelGroupId = singletonInstance.userProfile.getPubKey()
+
+      self.channelGroups[channelGroupId].chats.del(self.getChatIndex(channelGroupId, chatId))
```

Comment:
@alexandraB99 please use `delete` instead of `del` here, since delete keeps the order of the elements in the sequence.

Reply:
done!